### ISSUE TITLE
rocker/shiny:4.2.2 doesn't work on RHEL 7

### ISSUE THREAD

**Issue by wphampton** [rocker-org/rocker-versioned2#570]
### Container image name

rocker/shiny:4.2.2

### Container image digest

sha256:ccf6822a5955e35b9b365723d1b888c9d21f4eaa34089e10337c9734c758dd4c

### What operating system are you seeing the problem on?

Linux

### System information

 - Docker version 20.10.3, build 48d30b5
 - RHEL 7.9 Kernel 3.10.0-1160.76.1.el7.x86_64 on an x86_64

### Bug description

Receiving the following error:

[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 01_set_env: executing...
skipping /var/run/s6/container_environment/HOME
[cont-init.d] 01_set_env: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.

*** warning - no files are being watched ***
/opt/shiny-server/ext/node/bin/shiny-server[216]: ../src/node_platform.cc:61:std::unique_ptr<long unsigned int> node::WorkerThreadsTaskRunner::DelayedTaskScheduler::Start(): Assertion `(0) == (uv_thread_create(t.get(), start_thread, this))' failed.
 1: 0xb09980 node::Abort() [/opt/shiny-server/ext/node/bin/shiny-server]
 2: 0xb099fe  [/opt/shiny-server/ext/node/bin/shiny-server]
 3: 0xb78e9e  [/opt/shiny-server/ext/node/bin/shiny-server]
 4: 0xb78f66 node::NodePlatform::NodePlatform(int, v8::TracingController*) [/opt/shiny-server/ext/node/bin/shiny-server]
 5: 0xaccf49 node::InitializeOncePerProcess(int, char**, node::InitializationSettingsFlags, node::ProcessFlags::Flags) [/opt/shiny-server/ext/node/bin/shiny-server]
 6: 0xacd1f9 node::Start(int, char**) [/opt/shiny-server/ext/node/bin/shiny-server]
 7: 0x7fdc04abed90  [/lib/x86_64-linux-gnu/libc.so.6]
 8: 0x7fdc04abee40 __libc_start_main [/lib/x86_64-linux-gnu/libc.so.6]
 9: 0xa400ec  [/opt/shiny-server/ext/node/bin/shiny-server]

*** warning - no files are being watched ***
/opt/shiny-server/ext/node/bin/shiny-server[235]: ../src/node_platform.cc:61:std::unique_ptr<long unsigned int> node::WorkerThreadsTaskRunner::DelayedTaskScheduler::Start(): Assertion `(0) == (uv_thread_create(t.get(), start_thread, this))' failed.
 1: 0xb09980 node::Abort() [/opt/shiny-server/ext/node/bin/shiny-server]
 2: 0xb099fe  [/opt/shiny-server/ext/node/bin/shiny-server]
 3: 0xb78e9e  [/opt/shiny-server/ext/node/bin/shiny-server]
 4: 0xb78f66 node::NodePlatform::NodePlatform(int, v8::TracingController*) [/opt/shiny-server/ext/node/bin/shiny-server]
 5: 0xaccf49 node::InitializeOncePerProcess(int, char**, node::InitializationSettingsFlags, node::ProcessFlags::Flags) [/opt/shiny-server/ext/node/bin/shiny-server]
 6: 0xacd1f9 node::Start(int, char**) [/opt/shiny-server/ext/node/bin/shiny-server]
 7: 0x7f903118bd90  [/lib/x86_64-linux-gnu/libc.so.6]
 8: 0x7f903118be40 __libc_start_main [/lib/x86_64-linux-gnu/libc.so.6]
 9: 0xa400ec  [/opt/shiny-server/ext/node/bin/shiny-server]

### How to reproduce this bug?

```sh
docker run --rm -ti -p 3838:3838 rocker/shiny:4.2.2
```

**Comment by cboettig** [comment]
Thanks. I can't reproduce though.  It does look like your platform is a little dated -- are you able to test if you can reproduce this on a newer version of Docker (and/or maybe a newer linux kernel?)  

It is still working fine for me on ubuntu 22.04 with Docker engine 20.10.21

**Comment by wphampton** [comment]
Thank you for such a quick response. I will try today using latest Docker Desktop on Windows, but I trust it's working for you so that points to environment issues on my end.

Thank you for calling out our older version of Docker. I forgot to mention it but rocker/shiny:4.2.1 works fine, so I was not suspecting different behavior when I tried 4.2.2. I will ask our IT staff if they can upgrade that server. I'll report back results.

Thanks again,
Wes

**Comment by wphampton** [comment]
Closing this as I tried this same command on a very similar server which had a slightly newer version of Docker and it worked correctly. Below are the details of that successful server. There could be other things at play, but I'll just place it here in case it helps. Thanks for your help! -Wes

**Failed rocker/shiny:4.2.2**
Docker version 20.10.3, build 48d30b5
RHEL 7.9 Kernel 3.10.0-1160.76.1.el7.x86_64 on an x86_64

**Successful rocker/shiny:4.2.2**
Docker version 20.10.7, build f0df350
RHEL 7.9 Kernel 3.10.0-1160.76.1.el7.x86_64 on an x86_64
